src/__tests__/metrics.test.ts

```diff
@@ -8,21 +8,52 @@ const params = {
 }
 
 describe('metrics', () => {
-  test.each([[counter, Counter], [histogram, Histogram], [gauge, Gauge], [summary, Summary]])(
-    '%p passes correct params to %p',
-    (a, b) => {
-      a(params)
-      expect(b).toHaveBeenCalledWith(params)
-    }
-  )
+  test.each([
+    [counter, Counter],
+    [histogram, Histogram],
+    [gauge, Gauge],
+    [summary, Summary]
+  ])('%p passes correct params to %p', (a, b) => {
+    // @ts-expect-error (ts2339)
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    register.getSingleMetric.mockReturnValue(undefined)
+    a(params)
+    expect(b).toHaveBeenCalledWith(params)
+  })
+
+  test.each([[counter], [histogram], [gauge], [summary]])('prevents to register duplicate metrics, method=%p', (method) => {
```

Comment:
`prevents to register duplicate metrics`
to
`prevents registration of duplicate metrics`

Reply:
Will adjust this